.github/workflows/release.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           - windows-latest
           - macos-latest
         rust-version:
-          - nightly
+          - stable
```

Comment:
nit: I think it'd make sense to drop `rust-version` from the matrix completely. I believe `stable` channel is set by default.

Reply:
Good point!